### ISSUE TITLE
Missing HID macro definitions

### ISSUE THREAD

**Issue by obra** [NicoHood/HID#66]
If I revert a95361043, I'm now seeing missing definitions: 

```
/Users/jesse/Documents/Arduino/libraries/HID/src/SingleReport/BootKeyboard.cpp: In member function 'virtual bool BootKeyboard_::setup(USBSetup&)':
/Users/jesse/Documents/Arduino/libraries/HID/src/SingleReport/BootKeyboard.cpp:150:24: error: 'HID_REPORT_TYPE_FEATURE' was not declared in this scope
    if(setup.wValueH == HID_REPORT_TYPE_FEATURE){
                        ^
/Users/jesse/Documents/Arduino/libraries/HID/src/SingleReport/BootKeyboard.cpp:165:29: error: 'HID_REPORT_TYPE_OUTPUT' was not declared in this scope
    else if(setup.wValueH == HID_REPORT_TYPE_OUTPUT){
                             ^
/Users/jesse/Documents/Arduino/libraries/HID/src/SingleReport/BootKeyboard.cpp:173:29: error: 'HID_REPORT_TYPE_INPUT' was not declared in this scope
    else if(setup.wValueH == HID_REPORT_TYPE_INPUT)
                             ^
exit status 1

```

It looks almost like there's a missing header file that just never got checked in. Grepping for HID_REPORT_TYPE_ , I see checks but no definitions.

**Comment by NicoHood** [comment]
OH yeah, they did not merge my PR yet:
https://github.com/arduino/Arduino/pull/4105

Test it and comment that they should merge it.

**Comment by obra** [comment]
I'm confused about why the code change to add long recv support and the
new HID constants are in the same pull req. 


(I don't have anything that would want/need the long recv support)


On Wed, Dec 02, 2015 at 08:45:15AM -0800, Nico wrote:
>    OH yeah, they did not merge my PR yet:
>    [1]arduino/Arduino#4105
> 
>    Test it and comment that they should merge it.
> 
>    —
>    Reply to this email directly or [2]view it on GitHub.
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. Extend Usb recv control + some HID definitions
> 	https://github.com/arduino/Arduino/pull/4105
>    2. https://github.com/NicoHood/HID/issues/66#issuecomment-161360153
>    3. https://github.com/NicoHood/HID/issues/66#issuecomment-161360153

--

**Comment by NicoHood** [comment]
Both was needed for the RAWHID. And since they almost instantly merged all my PRs before, I thought they would do it now again, since the change is so minor. Comment there if its fine for you to merge.

**Comment by NicoHood** [comment]
Is it now working with the new hourly build? I still need to fix the >64 byte rawhid reads but it should compile fine still

**Comment by NicoHood** [comment]
Fixes with IDE 1.6.7 and 2.4.1 HID Project.
Could zou please check if it all works fine?
